medium/_01_fixed/answers/f_agg1.answer

```diff
@@ -1,5 +1,5 @@
 ===================================================
-class faculty1    max(ssn)    
+class joe.faculty1    max(ssn)    
 null     55     
```

Comment:
When I execute the query, the result is not null, so why is the result of TC null?
```
  class joe.faculty1       max(ssn)
===================================
  joe.faculty1                   55
```
Even if I execute it in develop, the result of TC is null.

Reply:
I don't know why null occurs in that TC. When first written, it is expected to have a purpose.

Reply:
Thank you. I hope it gets fixed later.